src/net/sourceforge/kolmafia/persistence/HolidayDatabase.java

```diff
@@ -278,7 +279,7 @@ public static int getHamburglarPosition(final ZonedDateTime dateTime) {
       return -1;
     }
 
-    var days = ChronoUnit.DAYS.between(dateTime, HolidayDatabase.COLLISION);
+    var days = Duration.between(HolidayDatabase.COLLISION, dateTime).toDays();
```

Comment:
@midgleyc what was the difference here?

Reply:
I read a stackoverflow post saying that ChronoUnit.DAYS.between is calendar days, but Duration.between is 24-hour days. We want the latter.

However that didn't matter for this case, what mattered is that the earlier date should come first, but it didn't here.